tests/integration/zipkin_integration_test.py

```diff
@@ -45,7 +76,58 @@ def _decode_binary_thrift_objs(obj):
     return spans
 
 
+def _decode_json_v1_span(obj):
```

Comment:
I don't think it's really necessary right now, but you could potentially use the jsonschema/swagger definition to build the span class and encoder/decoder programmatically.

I think it's likely that we'll end up going down this path for proto3...